Pi_java_stuff/runtimeDir/Selection.properties

```diff
@@ -0,0 +1,6 @@
+sauce1=Nutella
+sauce2=Marmelada
+sauce3=�alca Dreka
+sauce1Img=http://www.nutella.com/nutella-gl-theme/images/custom/Nutella.png
+sauce2Img=http://streamlinefoods.co.uk/wp-content/uploads/2014/02/Raspberry-Seedless-Jam_Streamline.png
```

Comment:
Ja vem, sj sm probavu ampak ni hotu odpret lokalnga patha. Mogoče če probam
z "file://path/to/image".